content/guides/additional/client.md

```diff
@@ -0,0 +1,2154 @@
++++
+title = "Clients"
```

Comment:
I would change this to a more precise/descriptive title. Idk what would be the corrent name for it. Globbed react/whatever frontends are also a case of "client" so it should be distinguishable from that

Reply:
mainly I'm thinking we want to frame it as this is an alternative to globs, http-api and eyre's channel system - it's *a* way to serve & interface rather than *the* way. Otherwise it might confuse FE developers who are used to react etc

Reply:
Going to try "Serving a Browser Game"